Dockerfile

```diff
@@ -0,0 +1,19 @@
+ARG BUILD_TAG=8-jdk-alpine
+ARG RUN_TAG=8-jre-alpine
+
+FROM openjdk:${BUILD_TAG} AS build-env
+WORKDIR /src
+
+# Copy src .dockerignore prevents copy of unnecessary files
+COPY . .
```

Comment:
Does this only copy the `src` folder? We also need files in the root folder and other nested folders (e.g. checkstyle) to perform the build.

Reply:
This copies the whole `root folder` of the project (`src folder`, `gradle` and others) ignoring the files mention on `.dockerignore` to the current folder (`WORKDIR`) of the container